src/Utils.jl

```diff
@@ -486,6 +494,14 @@ end
 function _flatten_results!(d::OrderedDict, results::Dict{String,Any}, prefix)
     if "times" in keys(results)
         d[prefix] = compute_summary_statistics(results)
+    elseif "samples" in keys(results) # This branch allows for Chairmarks.jl compatibility
```

Comment:
Should this be a separate PR? We can add a Chairmarks test for this branch then

Reply:
Oops! Yes. I committed that accidentally. I'll back it out of this PR. Thanks for spotting